src/list-grid.scss

```diff
@@ -0,0 +1,261 @@
+@import './new-settings';
+@import './mixins';
+
+/*!
+.fd-grid-list
+*/
+$block: #{$fd-namespace}-grid-list;
+
+.#{$block} {
+  $item: #{$block}__item;
+  $more: #{$block}__more;
+
+  $fd-grid-list-border-radius: var(--sapElement_BorderCornerRadius);
+  $fd-grid-list-active-text-color: var(--sapList_Active_TextColor);
+  $fd-grid-list-item-box-shadow: var(--sapContent_Shadow0);
+  $fd-grid-list-mode-box-shadow: 0 0 0 0.125rem var(--sapList_SelectionBorderColor);
+  $fd-grid-list-item-selected-background-color: var(--sapList_SelectionBackgroundColor);
+  $fd-grid-list-item-active-background-color: var(--sapList_Active_Background);
+  $fd-grid-list-item-hover-background-color: var(--sapList_Hover_Background);
+  $fd-grid-list-highlight: (
+    'success': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'error': ('backgroundColor': var(--sapErrorBorderColor)),
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  &__item {
+    @include fd-flex(column);
+    @include fd-fiori-focus();
+
+    background-color: var(--sapList_Background);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    height: 100%;
+    position: relative;
+
+    &--body {
+      @include fd-flex(column);
+
+      height: 100%;
+      justify-content: space-between;
+
+      &--padding {
+        padding: 1rem;
+      }
+    }
+
+    &--counter {
```

Comment:
<img width="419" alt="Screen Shot 2020-11-12 at 12 11 39 PM" src="https://user-images.githubusercontent.com/39598672/98972307-40996100-24e0-11eb-9603-fa87574d289d.png">

This is also not a modifier class. It would be a modifier class if you had `fd-grid-list__item fd-grid-list__item--counter`, then `fd-grid-list__item--counter` will modify the properties coming from `fd-grid-list__item`.
In your case it's a separate element and can be named `fd-grid-list__item-counter` or `fd-grid-list__counter` for simplicity. And you will have to add a reset mixin as it's a standalone element. Only modifier classes don't have reset, as the main class that they are modifying is bringing the reset.

Reply:
fixed